app/src/main/res/layout/activity_continuous_scan.xml

```diff
@@ -59,6 +59,21 @@
             android:visibility="gone"
             tools:visibility="visible" />
 
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            android:weightSum="100">
+
+        <View
+            android:id="@+id/quickView_swipeDetector"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="75"
+            android:layout_marginBottom="0dp"
+            />
+        </LinearLayout>
+
     </FrameLayout>
```

Comment:
Since you have added this inside the `FrameLayout` the with width and height as `match_parent`, now it won't be possible to click the buttons at the top, the overflow menu button and the flash button.